src/NuGet.Core/NuGet.Commands/Utility/MSBuildProjectFrameworkUtility.cs

```diff
@@ -143,15 +166,18 @@ public static class MSBuildProjectFrameworkUtility
             string targetPlatformIdentifier,
             string targetPlatformVersion,
             string targetPlatformMinVersion,
+            string clrSupport,
             bool isXnaWindowsPhoneProject,
             bool isManagementPackProject)
         {
+            bool isCppCli = clrSupport?.Equals("NetCore", StringComparison.OrdinalIgnoreCase) == true;
```

Comment:
Don't we need to check if the project extension is `.vcxproj` also?

Let's say if I create a C# project and add `<CLRSupport>NetCore</CLRSupport` property then `isCppCli` will be `true`. It is very much possible that I am missing something :)

Reply:
I guess we could. 
Don't think it makes too big of a difference, but I can check if it you'd like me to.

Reply:
If it takes less time around 10-15 mins, can you please check that? I just want to make sure that the method returns expected results for non-C++ projects.

Reply:
I wouldn't expect there are *any* consequences here. 
Having a csproj setting CLRSupport is not really something anyone would do. 
I'll make the change.